modules/adding-node-iso-configs.adoc

```diff
@@ -112,13 +112,22 @@ This path is also used to store the generated artifacts.
 |The name of the generated output image.
 |String
 
+|`--pxe`
+|Generates PXE assets instead of a bootable ISO file.
+|None
+
 |`-a`, `--registry-config`
 |The path to your registry credentials.
 Alternatively, you can specify the `REGISTRY_AUTH_FILE` environment variable.
 The default paths are `${XDG_RUNTIME_DIR}/containers/auth.json`, `/run/containers/${UID}/auth.json`, `${XDG_CONFIG_HOME}/containers/auth.json`, `${DOCKER_CONFIG}`, `~/.docker/config.json`, `~/.dockercfg.`
 The order can be changed through the deprecated `REGISTRY_AUTH_PREFERENCE` environment variable to a "docker" value, in order to prioritize Docker credentials over Podman.
 |String
 
+|`-r`, `--report`
+|Generates a report of the node creation process regardless of whether the process is successful or not.
+Without specifying this flag, reports are only generated in cases of failure.
+|Boolean
```

Comment:
Is this value correct? Or do you just specify the flag without a value?

Reply:
For a bool flag, it presence automatically sets it to true, so `--report` or `--report=true` are the same

Reply:
Makes sense, thanks for clearing that up!